packages/execute-driver/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@appium/execute-driver-plugin",
+  "version": "1.0.0",
+  "description": "Plugin for batching and executing driver commands with Appiums",
+  "main": "./build/index.js",
```

Comment:
given the horror over in `appium`, you should probably just create a root `index.js` wrapper which requires `./build`)

Reply:
good idea, maybe we can do that in one PR that brings this whole repo into line with the way we do appium/appium.